packages/vscode-ui5-language-assistant/test/test-fixtures/validations/flex-enabled-spec/manifest.json

```diff
@@ -0,0 +1,31 @@
+{
+  "_version": "1.12.0",
+  "sap.app": {
+    "id": "testFixure",
+    "type": "application",
+    "applicationVersion": {
+      "version": "1.0.0"
+    },
+    "title": "{{appTitle}}"
+  },
+
+  "sap.ui": {
```

Comment:
is `"sap.ui"` section mandatory in the manifest.json?

Reply:
yes